src/contexts/ThemeContext/ThemeContext.jsx

```diff
@@ -0,0 +1,43 @@
+import { useContext, createContext, useState, useEffect } from "react";
+
+const defaultTheme = { theme: "light" };
+
+const ThemeContext = createContext(defaultTheme);
+
+const ThemeProvider = ({ children }) => {
+  const [theme, setTheme] = useState("light");
+  const Toggle = () => {
+    if (theme === "dark") {
+      setTheme("light");
+      document.body.classList.remove("dark-theme");
+      document.body.classList.add("light-theme");
+      localStorage.setItem("theme", "light");
+    } else {
+      setTheme("dark");
+      document.body.classList.remove("light-theme");
+      document.body.classList.add("dark-theme");
```

Comment:
For dark mode you can look for the better approach

Reply:
Ok, I will check that